lazy-imports-import.html

```diff
@@ -8,6 +8,7 @@
 subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
 -->
 <script>
+  'use strict';
```

Comment:
Yes, it should. I have been writing in ES6 and haven't actually used `'use strict'` for a while. Good catch, thanks!